spec/controllers/videos_controller.rb

```diff
@@ -0,0 +1,114 @@
+require 'spec_helper'
+
+describe VideosController do
+  let!(:video) { Fabricate(:video, title:"The God Father") }
+
+  describe "GET show" do
+    context "for authecticated user" do
+      before do
+        session[:user_id] = Fabricate(:user).id
+        get :show, { id: video.id }
+      end
+
+      it "sets the @video variable for authenticated user" do
```

Comment:
Your context is "for authenticated user", you don't need to repeat it in your spec names.